runtime/lookup.c

```diff
@@ -625,7 +625,7 @@ lookupFindTable(uchar *name)
 {
 	lookup_ref_t *curr;
 
-	for(curr = loadConf->lu_tabs.root ; curr != NULL ; curr = curr->next) {
+	for(curr = loadConf->lu_tabs.last ; curr != NULL ; curr = curr->next) {
```

Comment:
Isn't ```last``` actually pointing at the last node, so that you can find at most that one? Without looking at the full code, ```root``` sounds right to me, which is also backed by the testbench failures. It would be great if you could provide additional information, best in the commit, why this change is needed.